app/frontend/src/components/designer/FormViewer.vue

```diff
@@ -331,9 +331,12 @@ export default {
         for (let property in obj) {
           const innerObject = obj[property];
 
-          // When the form contains a Data Grid there will be an array that
-          // needs to be checked, and an array of properties to be unset.
-          if (Array.isArray(innerObject)) {
+          if (propNeeded === property) {
+            fields = fields + stack + '.' + property;
+            return fields.replace(/^\./, '');
```

Comment:
The problem where the multiselects always being copied is because both datagrids and multiselects appear as Arrays. The fix was to move the `propNeeded === property` clause to come before the Array check for datagrids.